elasticsearch/values.yaml

```diff
@@ -159,3 +159,6 @@ ingress:
 
 nameOverride: ""
 fullnameOverride: ""
+
+sysctlInitContainer:
+  enabled: false
```

Comment:
Did you mean to put false here? The default value that you added to the test is `true` so the template tests are going to fail.